### PR TITLE
add a sentence to hindreen from my branch

### PR DESCRIPTION
**Pull request by Hano1388**

### CODE REVIEW

File: hindreen.md [hunk 1]
```diff
@@ -1 +1,3 @@
 ### Hindreen
+
+just a sentence
```
Comment (davidko604) [inline hindreen.md:4]: This line is too short

Comment (Hano1388) [inline hindreen.md:4]: just add a strong tag to it

**Review by aj-developer2020 — COMMENTED** [review]
looks good to me

**Review by davidko604 — APPROVED** [review]

**Review by luke-bo — APPROVED** [review]
perf

**Review by dbtee — COMMENTED** [review]
idklol

**Pull request merged**